docs/source/checks/lda.rst

```diff
@@ -0,0 +1,237 @@
+
+    This parameter expects an int and controls the maximum number of iterations
+    the LDA model uses before it finishes. Setting it higher may lead to longer
+    training times but also potentially better results.
+
+    The default value is 10.
+
+``lib``:
+    This parameter is only passed to the pre-processing :ref:`lda_embedding`.
+
+    This parameter expects either the string 'sklearn' or 'gensim', as these
+    currently are the only two supported LDA libraries. In some cases one may
+    perform better than the other, so at best both are executed together.
+
+    However for the sake of simplicity and because it is the more known library,
+    the default value is 'sklearn'.
+
+``random_state``:
+    This parameter is only passed to the pre-processing :ref:`lda_embedding`.
+
+    This parameter expects an integer and is used to produce deterministic results.
+
+    The default value is 0.
+
+``trained_model``:
+    This parameter is only passed to the pre-processing :ref:`lda_embedding`.
+
+    This parameter expects a pre-trained LDA model.
+    The default value is None which means no trained model is provided and instead
+    a new model is trained. If a pre-trained model is given it will not be retrained!
+    That's why it is highly discouraged to run the check on more than one column with
+    ``trained_model`` set!
+
+``stop_words``:
+    This parameter is only passed to the pre-processing :ref:`lda_embedding`.
+
+    This parameter expects a string of a languages or a list with multiple strings
+    that represent languages. The given language(s) is/are used to determine a list
+    stop words that should be filtered out before the text is processed.
+    The default value is 'english.
+
+``max_features``:
+    This parameter is only passed to the pre-processing :ref:`lda_embedding`.
+
+    This parameter expects an integer and it limits how many features are used when
+    using the :ref:`count_vectorizer` to calculate the document term matrices.
+    If its value is None, all features of both datasets are used.
+
+    The default value is None.
+
+``word_clouds``:
+    This parameter expects a boolean. It indicates whether word clouds should be
+    displayed in the visualization as they can lead to problems in some environments
+    that cannot install third-party libraries that themselves rely on different
+    third-party libraries.
+
+    The default value is True.
+
+``ldavis``:
+    This parameter expects a boolean. It indicates whether an interactive pyLDAvis
+    plot should be provided as it is relatively ressource-intensive to compute.
+
+    The default value is True.
+
+
+Implementation
+--------------
+
+Algorithm
++++++++++
+
+The :ref:`LdaCheck` works as follows:
+
+1.  First, calculate the LDA embeddings for all specified text columns of the two datasets df1 and df2.
+2.  Then, take the percentual difference between each topic for each column, so *diff_topic = topic(df1) - topic(df2)*
```

Comment:
This is not clear enough. Can you explain the individual steps and include a frequency table?

Reply:
I think this is would be too detailed in comparison to our other docs